app/main.py

```diff
@@ -1 +1,37 @@
-# write your code here
+class Animal:
+    animal = []
```

Comment:
The class attribute `animal` is defined but not used. It seems like you intended to use `alive` instead. Consider renaming `animal` to `alive`.